inventory/packages/yum.go

```diff
@@ -31,7 +31,8 @@ var (
 	yumInstallArgs       = []string{"install", "--assumeyes"}
 	yumRemoveArgs        = []string{"remove", "--assumeyes"}
 	yumCheckUpdateArgs   = []string{"check-update", "--assumeyes"}
-	yumUpdateArgs        = []string{"update", "--assumeno", "--cacheonly"}
+	yumUpdateArgs        = []string{"update", "--assumeyes"}
+	yumUpdatesArgs       = []string{"update", "--assumeno", "--cacheonly"}
 	yumUpdateMinimalArgs = []string{"update-minimal", "--assumeno", "--cacheonly"}
```

Comment:
my understanding is we are using --assumeno as an alternative to list command. consider renaming the variables to reflect the listing behavior. yumlistupdatesargs?